role-service-client/src/test/java/uk/ac/ox/ndph/mts/roleserviceclient/CreateEntityTest.java

```diff
@@ -31,7 +31,7 @@
     public static MockWebServerWrapper webServer;
     private static final TestClientBuilder builder = new TestClientBuilder();
     private RoleServiceClient roleServiceClient;
-    private Consumer<HttpHeaders> authHeaders = RoleServiceClient.noAuth();
+    private String token = "some-token";
```

Comment:
```suggestion
    private String token = "some-token";
    private Consumer<HttpHeaders> authHeaders = RoleServiceClient.bearerAuth(token);
```
Then we can replace all RoleServiceClient.bearerAuth(token) with authHeaders